origin_ui/src/app/(dashboard)/page.tsx

```diff
@@ -31,7 +32,10 @@ export default function Home() {
     return (
         <Box width={"100%"}>
             <Grid container spacing={2}>
-                <Grid item xs={12} lg={6}>
+                <Grid item xs={12} lg={4}>
+                    <StatusBox/>
+                </Grid>
+                <Grid item xs={12} lg={8}>
                     <Box sx={{backgroundColor: "#F6F6F6", borderRadius: "1rem"}} p={2}>
```

Comment:
* It can be very helpful if we can display the error messages for the the critical and warning status.
* We might want to have different colors for the different status (critical/warning)
* The color fill is a bit heavy for the user reading the dashboard for OK and warning status. A colored-border box might be more user friendly?
* Capitalize the first letter of the status?